libs/blocks/merch-card/merch-card.css

```
@@ -0,0 +1,350 @@
+/* stylelint-disable selector-class-pattern */
+.merch-card.segment-blade {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  min-height: 222px;
+  width: 100%;
+  flex: 1;
+  min-width: 378px;
+  max-width: 378px;
+}
+
+.consonant-CardsGrid--oneup .merch-card.consonant-Card {
+  max-width: 100%;
+}
+
+.merch-card.consonant-Card .consonant-CardFooter {
+  padding-top: 0;
+}
+
+.consonant-CardsGrid--2up .merch-card {
+  width: 378px;
+}
+
+.merch-card .consonant-CardFooter-row {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+  align-items: center;
+  height: 32px;
+  white-space: nowrap;
+}
+
+.merch-card [data-cells="1"]:not(.consonant-CardFooter-row--fluid) .consonant-CardFooter-cell {
+  max-width: fit-content;
+  display: flex;
+  margin-left: var(--spacing-xs);
+}
+
+.consonant-CardsGrid--4up .merch-card [data-cells="1"]:not(.consonant-CardFooter-row--fluid) .consonant-CardFooter-cell {
+  margin-left: var(--spacing-xxs);
+}
+
+.merch-card .consonant-MerchCard-ProductIcon {
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: relative;
+  margin-right: var(--spacing-xxs);
+  margin-bottom: var(--spacing-xxs);
+  display: block;
+  float: left;
+  width: 40px;
+  height: 40px;
+}
+
+.merch-card .consonant-CardFooter .consonant-CardFooter-row .button--inactive {
+  display: none;
+}
+
+.merch-card.special-offers,
+.merch-card.segment,
+.merch-card.plans {
+  border-radius: 16px;
+}
+
+.merch-card .consonant-ProductCard-inner,
+.merch-card .consonant-SegmentBlade-inner,
+.merch-card .consonant-SpecialOffers-inner,
+.merch-card .consonant-PlansCard-inner {
```

```suggestion
.merch-card div[class$="-inner"] {
```
Just a thought, you could use this to check end of class pattern instead of listing all specific types. Same as below line 91.